src/redux/backend-session.ts

```diff
@@ -289,9 +306,11 @@ export const epicStartSession: Epic<Action, State> = (action$, store) => {
       const forceNew =
         startSessionAction.payload.force_new ||
         selectBackendSessionCountPostNewSession(store.getState()) === 1;
+      const imageTag = selectImageTag(startSessionAction.payload.language, selectLangVersion(store.getState()));
```

Comment:
It's not a good practice to rely on your store in your actions. Ideally, the language version is part of your action payload.